Storage/container/one_to_one.h

```diff
@@ -189,22 +255,29 @@ class GenericOneToOne {
   iterator_t end() const { return iterator_t(map_.end()); }
 
  private:
+  void DoAddWithLastModified(std::chrono::microseconds us, const key_t& key, const T& object) {
+    auto& placeholder = map_[key];
+    placeholder = std::make_unique<T>(object);
+    forward_[key.first] = placeholder.get();
+    transposed_[key.second] = placeholder.get();
+    last_modified_[key] = us;
+  }
+
   void DoErase(const key_t& key) {
     forward_.erase(key.first);
     transposed_.erase(key.second);
     map_.erase(key);
   }
```

Comment:
Or, assuming you thought of it more than I did and my comment is misleading, rename `DoErase` into some `DoEraseFromMap` / `DoEraseWithoutTouchingLastModified`, etc.